assembl/static2/js/app/components/styleguide/icons.jsx

```diff
@@ -98,13 +126,19 @@ class Icons extends React.Component {
         <section>
           <h3 className="dark-title-3">Code</h3>
           <pre>
-            &lt;span className="assembl-icon-add black"&gt;&lt;/span&gt;
+            &lt;span className="assembl-icon-down-dir color"&gt;&lt;/span&gt;
```

Comment:
It's in the style guide, we want to show the source code in a pre tag, so you need to escape it I guess.